titus_isolate/api/status.py

```diff
@@ -19,12 +22,30 @@ def set_em(event_manager):
     __event_manager = event_manager
 
 
+@app.route('/isolate/<workload_id>')
+def isolate_workload(workload_id, timeout=None):
+    if timeout is None:
```

Comment:
Should I pass a value for timeout as a param? or do you always want to look at the FP.